packages/frontend/src/components/SchedulersView/SchedulersView.tsx

```diff
@@ -93,9 +105,7 @@ const Schedulers: FC<SchedulersProps> = ({
                                                 <Text color="gray.5">
                                                     Schedule type:{' '}
                                                     <Text color="white" span>
-                                                        {item.format === 'csv'
-                                                            ? 'CSV'
-                                                            : 'Image'}
+                                                        {format()}
```

Comment:
how come it's a function?

Reply:
previously it was a `format===csv?csv:image` now it needs to be csv, image or google sheets, I find a function with return more readable.